src/Elsa.CustomActivities/Activities/QuestionScreen/Question.cs

```diff
@@ -56,23 +56,25 @@ public class Question
         public string QuestionHint { get; set; } = null!;
 
         [HeActivityInput(Hint = "Question guidance",
-            UIHint = HePropertyUIHints.MultiLine,
+            UIHint = HePropertyUIHints.TextGroup,
+            DefaultSyntax = TextActivitySyntaxNames.TextGroup,
+            ExpectedOutputType = ExpectedOutputHints.TextGroup,
             ConditionalActivityTypes = new[]
             {
-                QuestionTypeConstants.CurrencyQuestion,
-                QuestionTypeConstants.DecimalQuestion,
-                QuestionTypeConstants.IntegerQuestion,
-                QuestionTypeConstants.PercentageQuestion,
-                QuestionTypeConstants.CheckboxQuestion,
-                QuestionTypeConstants.RadioQuestion,
-                QuestionTypeConstants.PotScoreRadioQuestion,
-                QuestionTypeConstants.DateQuestion,
-                QuestionTypeConstants.TextQuestion,
-                QuestionTypeConstants.TextAreaQuestion,
-                QuestionTypeConstants.WeightedCheckboxQuestion,
-                QuestionTypeConstants.WeightedRadioQuestion
+                        QuestionTypeConstants.CurrencyQuestion,
+                        QuestionTypeConstants.DecimalQuestion,
+                        QuestionTypeConstants.IntegerQuestion,
+                        QuestionTypeConstants.PercentageQuestion,
+                        QuestionTypeConstants.CheckboxQuestion,
+                        QuestionTypeConstants.RadioQuestion,
+                        QuestionTypeConstants.PotScoreRadioQuestion,
+                        QuestionTypeConstants.DateQuestion,
+                        QuestionTypeConstants.TextQuestion,
+                        QuestionTypeConstants.TextAreaQuestion,
+                        QuestionTypeConstants.WeightedCheckboxQuestion,
+                        QuestionTypeConstants.WeightedRadioQuestion
             })]
-        public string QuestionGuidance { get; set; } = null!;
```

Comment:
I thought the plan was to keep QuestionGuidance in, to handle the legacy occurrences of it (Just removing the HeActivityInput Attribute)?  Or is that handled in a different way now?